components/molecules/ComponentDateFilter/component-date-filter.tsx

```diff
@@ -22,15 +24,27 @@ const ComponentDateFilter = ({ setRangeFilter, defaultRange }: ComponentDateFilt
   return (
     <div className="flex items-center text-sm bg-white rounded-lg shrink-0 w-max">
       {dates.map((range, index) => (
-        <div
-          onClick={() => handleFilterClick(range)}
-          className={`px-4 py-1.5 rounded-lg cursor-pointer transition text-light-slate-9 ${
-            activeFilter === range && "border text-light-slate-12 bg-light-slate-5"
-          }`}
+        <Tooltip
           key={index}
+          content={`
+          ${rangeFormatter(range)}
+          ${range === 7 ? "from today" : "from the last commit"}
+        `}
+          direction="top"
         >
-          {rangeFormatter(range)}
-        </div>
+          <span className="sr-only">
+            {rangeFormatter(range)} {rangeFormatter(range)}${range === 7 ? "from today" : "from the last commit"}
+          </span>
```

Comment:
I made a mistake suggesting the label here. The button already has text. We can improve the accesibility in another PR to describe what the range does. I'll create a separate issue for that.

Reply:
Remove the label I suggested, and I think this is good to go @OgDev-01.

Reply:
I created #212 for this.